webpack.dev.js

```diff
@@ -1,34 +1,40 @@
 const path = require('path');
+const webpack = require('webpack');
 
 module.exports = {
-    mode: 'development',
-    entry: path.join(__dirname, '/lib'),
-    resolve: {
-        extensions: ['.js']
-    },
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                use: {
-                    loader: 'babel-loader'
-                }
-            },
-            {
-                test: /\.js$/,
-                loader: 'string-replace-loader',
-                options: {
-                    multiple: [
-                        { search: 'api.dreamdata.cloud/v1', replace: 'localhost:8080/v1' },
-                        { search: `'https://'`, replace: `'https://'` }
-                    ]
-                }
-            }
-        ]
-    },
-    output: {
-        library: 'dreamdata',
-        libraryTarget: 'umd',
-        filename: 'dreamdata.js'
-    }
+  mode: 'development',
+  entry: path.join(__dirname, '/lib'),
+  resolve: {
+    extensions: ['.js']
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: {
+          loader: 'babel-loader'
+        }
+      },
+      {
+        test: /\.js$/,
+        loader: 'string-replace-loader',
+        options: {
+          multiple: [
+            { search: 'api.dreamdata.cloud/v1', replace: 'localhost:8080/v1' },
+            { search: `'https://'`, replace: `'https://'` }
```

Comment:
Search `https://` and replace with `https://` - why do we need this?

Reply:
typo - supposed to replace to http, fixed in 4.0.1